draft-ietf-dnsop-svcb-https.md

```diff
@@ -799,9 +774,9 @@ The "port" SvcParamKey defines the TCP or UDP port
 that should be used to contact this alternative service.
 If this key is not present, clients SHALL use the origin server's port number.
 
-The presentation format of the SvcParamValue is a numeric value
-between 0 and 65535 inclusive.  Any other values (e.g. the empty value)
-are syntax errors.
+The presentation `value` of the SvcParamValue is a single decimal number
+between 0 and 65535 in ASCII.  Any other `value` (e.g. an empty value)
+is a syntax error.
```

Comment:
It would be useful to never have to apply escape processing to port so append.  "No escape processing should be applied to the port value."

Reply:
Done